src/html5_parser/parser/document.rs

```diff
@@ -478,23 +491,55 @@ mod tests {
             _ => panic!(),
         }
 
-        let _ = document.get_mut().add_node(node1, NodeId(0));
-        let _ = document.get_mut().add_node(node2, NodeId(1));
+        let _ = document.get_mut().add_node(node1, NodeId::from(0));
+        let _ = document.get_mut().add_node(node2, NodeId::from(1));
```

Comment:
`0.into()` and `1.into()` will also work here, btw.  Although I think `NodeId::from(0)`, etc., is good.